src/platform/forms-system/test/js/review/ReviewCollapsibleChapter.unit.spec.jsx

```diff
@@ -375,6 +377,113 @@ describe('<ReviewCollapsibleChapter>', () => {
     });
   });
 
+  it('should show single page title when != chapter title', () => {
+    const testPageTitle = 'test page title';
+    const testChapterTitle = 'test chapter title';
+
+    const onEdit = sinon.spy();
+    const pages = [
+      {
+        pageKey: 'test',
+        title: testPageTitle,
+        updateFormData: (oldData, newData) => ({ ...newData, bar: 'baz' }),
+      },
+    ];
+    const chapterKey = 'test';
+    const chapter = {
+      title: 'test chapter title',
+    };
+    const form = {
+      pages: {
+        test: {
+          title: testPageTitle,
+          schema: {
+            type: 'object',
+            properties: {
+              foo: { type: 'string' },
+            },
+          },
+          uiSchema: {},
+          editMode: false,
+        },
+      },
+      data: {},
+    };
+    const setPagesViewed = sinon.spy();
+
+    const wrapper = mount(
+      <ReviewCollapsibleChapter
+        setPagesViewed={setPagesViewed}
+        viewedPages={new Set()}
+        onEdit={onEdit}
+        open
+        expandedPages={pages}
+        chapterKey={chapterKey}
+        chapterFormConfig={chapter}
+        form={form}
+      />,
+    );
+
+    const titleDiv = wrapper.find('.form-review-panel-page-header');
+
+    expect(titleDiv.length).to.equal(1);
+    expect(titleDiv.text()).to.equal(testPageTitle);
+    expect(titleDiv.text()).to.not.equal(testChapterTitle);
+
+    wrapper.unmount();
+  });
+
+  it('should not show single page title when equals chapter title', () => {
+    const testChapterTitle = 'test chapter title';
+
+    const onEdit = sinon.spy();
+    const pages = [
+      {
+        pageKey: 'test',
+        title: testChapterTitle,
+        updateFormData: (oldData, newData) => ({ ...newData, bar: 'baz' }),
+      },
+    ];
+    const chapterKey = 'test';
+    const chapter = {
+      title: testChapterTitle,
+    };
+    const form = {
+      pages: {
+        test: {
+          title: testChapterTitle,
+          schema: {
+            type: 'object',
+            properties: {
+              foo: { type: 'string' },
+            },
+          },
+          uiSchema: {},
+          editMode: false,
+        },
+      },
+      data: {},
+    };
+    const setPagesViewed = sinon.spy();
+
+    const wrapper = mount(
+      <ReviewCollapsibleChapter
+        setPagesViewed={setPagesViewed}
+        viewedPages={new Set()}
+        onEdit={onEdit}
+        open
+        expandedPages={pages}
+        chapterKey={chapterKey}
+        chapterFormConfig={chapter}
+        form={form}
+      />,
+    );
+
+    expect(wrapper.exists('form-review-panel-page-header')).to.be.false;
```

Comment:
What was happening when you checked for a zero-length?

```js
const titleDiv = wrapper.find('.form-review-panel-page-header');

expect(titleDiv.length).to.equal(0);
```

Reply:
AssertionError: expected 1 to equal 0

but 

    wrapper.find('.form-review-panel-page-header').text()

prints out "target Element not found"

Reply:
Fixed test